web/src/main/java/org/openmrs/module/web/WebModuleUtil.java

```diff
@@ -775,7 +775,7 @@ public static void stopModule(Module mod, ServletContext servletContext, boolean
 			if (root.getElementsByTagName("dwr").getLength() > 0) {
 				
 				// get the dwr-module.xml file that we're appending our code to
-				File f = new File(realPath + "/WEB-INF/dwr-modules.xml".replace("/", File.separator));
+				File f = new File(realPath, "WEB-INF/dwr-modules.xml".replace("/", File.separator));
```

Comment:
same as above, the goal of this ticket is to get rid of all the manual path constructions in the code using `File.separator` or `System.getProperty("file.separator")` and `File.separatorChar`